src/tree/SubscriptionTreeItem.ts

```diff
@@ -117,7 +117,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         await detectDockerfile(context);
 
         if (context.dockerfilePath) {
-            promptSteps.push(new SiteNameStep("containerizedfunctionApp"));
+            promptSteps.push(new SiteNameStep("containerizedFunctionApp"));
```

Comment:
I feel like it's more stylish to just use a ternary here
`promptSteps.push(new SiteNameStep(context.dockerfilePath  ? "containerizedFunctionApp" : "functionApp"));`